src/Environments/Windows/win32characteristics.xml

```diff
@@ -29,7 +29,7 @@
   </entry>
   <entry name="ExitProcess">
     <characteristics>
-      <terminates>true</terminates>
+      <terminates>false</terminates>
```

Comment:
This required to ensure a complete match is made, no change

Reply:
This needs to be changed to `true`; otherwise you will be breaking the parts of Reko's data flow analysis that depend on terminating functions being reported as such. I would suggest that you create a unit test to indicate where this breaks the "complete match" you mentioned in your comment.